### PR TITLE
pages/screens- rewrite screens page

### PR DESCRIPTION
**Pull request by junhoyeo**

### CODE REVIEW

File: web/.babelrc
Comment (softmarshmallow) [inline web/.babelrc]: I should have asked this on previous PR.
does next js requires explicit babelrc configuration?

https://nextjs.org/docs/advanced-features/customizing-babel-config

**Review by softmarshmallow — APPROVED** [review]

File: web/.babelrc
Comment (junhoyeo) [reply web/.babelrc]: Please wait, I'll double-check the environment so we can reduce presets

Comment (junhoyeo) [reply web/.babelrc]: I just recognized that we can reuse `next/babel` like the [docs](https://nextjs.org/docs/advanced-features/customizing-babel-config).
I'll fix it, but please mind that we still need the `.babelrc` for linaria(extracting the CSS at build time)

Comment (softmarshmallow) [reply web/.babelrc]: Alright, after next commit, you may merge this yourself.

**Comment by junhoyeo** [comment]
Oops, I forgot to remove the unused babel dependencies. Don't merge yet.

File: web/package.json
Comment (junhoyeo) [inline web/package.json]: I'll check if this can be removed

File: web/pages/screens/index.tsx
Comment (junhoyeo) [inline web/pages/screens/index.tsx:38]: We should improve this by providing flexible typings in `SceneItem` someday

**Pull request merged**
